extensions/rich_text_components/components.py

```diff
@@ -142,7 +142,16 @@ class Link(BaseRteComponent):
 class Math(BaseRteComponent):
     """Class for Math component."""
 
-    pass
+    @classmethod
+    def validate(cls, value_dict):
+        """Validates Math component."""
+        super(Math, cls).validate(value_dict)
+        filename_pattern_regex = (
+            r'mathImg_[0-9]+_\S{10}_height_[0-9d]+_width_[0-9d]+_vertical_['
+            '0-9d]+.(svg)$')
+        filename = value_dict['math_content-with-value']['svg_filename']
```

Comment:
Do we need to value_dict['math_content-with-value'] to be a dict?
Do we need to validate `svg_filename` in value_dict['math_content-with-value']? (OR is it already validate in base class)
Do we need to validate other properties of value_dict['math_content-with-value']?

Reply:
Yeah the structure/schema of `value_dict['math_content-with-value` is validated in the base class(`objects.MathExpressionContent`) itself.
We don't need any-more validations